sdk/openai/azure-ai-openai-assistants/src/test/java/com/azure/ai/openai/assistants/AzureRetrievalAsyncTest.java

```diff
@@ -32,14 +34,15 @@ public class AzureRetrievalAsyncTest extends AssistantsClientTestBase {
 
     AssistantsAsyncClient client;
 
+    @Disabled("Retrieval tools are not supported in Azure")
```

Comment:
@jpalvarezl  I added Disabled annotation only to `basicRetrieval` test when testing in AZURE resource

Reply:
Yikes! That's unfortunate, but it wouldn't be the first test we had to disable for this reason. Thanks @mssfang !